src/RPC.js

```diff
@@ -237,7 +237,7 @@ class RPC {
                 offset = this.Wallet.Utils.isHex(offset)
             }
             let DataStoreUTXOIDs = [];
-            while (true) {
+            while (constant.ReqCondition) {
```

Comment:
`constant.ReqCondition` here addresses the rule https://eslint.org/docs/rules/no-constant-condition